README.md

```diff
@@ -1911,7 +1911,7 @@ sudo sysdiagnose -f ~/Desktop/
 #### Create Bootable Installer
 ```sh
 # macOS 11 (Big Sur)
-sudo /Applications/Install\ macOS\ Big\ Sur.app/Contents/Resources/createinstallmedia --volume /Volumes/USB
+sudo /Applications/Install\ macOS\ Big\ Sur.app/Contents/Resources/createinstallmedia --volume /Volumes/USB --nointeraction --downloadassets
```

Comment:
Again: have you verified that those flags do indeed work?

Reply:
Yes, they so actually: 
```
➜  ~ sudo /Applications/Install\ macOS\ Big\ Sur.app/Contents/Resources/createinstallmedia --help
Usage: createinstallmedia --volume <path to volume to convert>

Arguments
--volume, A path to a volume that can be unmounted and erased to create the install media.
--nointeraction, Erase the disk pointed to by volume without prompting for confirmation.
--downloadassets, Download on-demand assets that may be required for installation.

Example: createinstallmedia --volume /Volumes/Untitled
➜  ~
```

Reply:
Thanks for confirming. Due diligence like this keeps this list from devolving into just another collection of half-broken lazy copy-pasting.